sources/api/host-containers/src/main.rs

```diff
@@ -363,14 +363,8 @@ where
         name,
         field: "source",
     })?;
-    let enabled = image_details.enabled.context(error::MissingField {
-        name,
-        field: "enabled",
-    })?;
-    let superpowered = image_details.superpowered.context(error::MissingField {
-        name,
-        field: "superpowered",
-    })?;
+    let enabled = image_details.enabled.unwrap_or(false);
```

Comment:
We talked a bit offline - it seems like we might be able to use serde default= annotations on the fields in the model, instead of doing it here in code.  That would mean that the defaults (false) show up when you query settings, rather than leaving users in suspense and having to check docs.

Reply:
After some epic testing, it seems the serde default is applying too strongly and overwriting things from our defaults.toml or even user data.  We can figure that out separately someday!